lib/testutils/src/lib.rs

```diff
@@ -142,6 +142,12 @@ impl TestEnvironment {
         }
     }
 
+    /// Panic if `CI` environment variable is set to a non-empty value
+    pub fn ensure_running_outside_ci(reason: &str) {
```

Comment:
nit: move to module-level function because this does nothing related to `TestEnvironment`.